Sources/WebDriver/ScreenOrientation.swift

```diff
@@ -0,0 +1,5 @@
+// Screen orientations
+public enum ScreenOrientation: String, Codable {
+    case portrait = "PORTRAIT"
+    case landscape = "LANDSCAPE"
+}
```

Comment:
Missing new line at the end of the file.  Why the all uppercase for the values?

Reply:
They're all caps in the docs [here](https://www.selenium.dev/documentation/legacy/json_wire_protocol/#sessionsessionidorientation). Looks like it's a leftover from the Java implementation